wiki_indexer/connector.py

```diff
@@ -0,0 +1,94 @@
+from sqlalchemy import create_engine, MetaData, Table, engine
+from elasticsearch import Elasticsearch
+import logging
+from multiprocessing.pool import ThreadPool
+import configparser
+import sqlalchemy
+
+
+class DatabaseConnectionError(Exception):
+    pass
+
+
+class ElasticConnectionError(Exception):
+    pass
+
+
+class Connector:
+    es = Elasticsearch()
+    ROOT_LOGGER = logging.getLogger()
+    ROOT_LOGGER.addHandler(logging.FileHandler('log.txt'))
```

Comment:
I assume to initialize handler in the main() function (nevertheless outside from this module)